### ISSUE TITLE
Question: Definition of rps

### ISSUE THREAD

**Issue by eaheah** [hgrecco/pint#543]
We looked at the default_en unit definitions, and we couldn't find rps defined except in terms of hertz. However, the assumption we came in with was that rps should convert to rpm, and thus, to radians/second instead of 1/second. Can you clarify why the units were defined this way?

# Frequency
[frequency] = 1 / [time]
hertz = 1 / second = Hz = rps
revolutions_per_minute = revolution / minute = rpm
counts_per_second = count / second = cps

# Angle
turn = 2 * pi * radian = revolution = cycle = circle

**Comment by hgrecco** [comment]
This is indeed inconsistent and should be fixed. We should do this. But be aware that this does not solve all problems. It goes much deeper. See #63, #182 and a possible solution in the last comment in #505

**Comment by kbauer** [comment]
Possibly related, so I write it into this ticket for now.

The definition of `hertz` may be incorrect in `Pint 0.17`.

The convention that I know from Physics and engineering contexts, is:
  - "Angular frequencies" = "angular rates" are simply "angle per second", typially written as `1/s` in SI units.
  - "Frequencies" are given as "cycles per second" and "Hertz" is used as the unit. One cycle per second is a frequency of one Hertz. In SI units, this too is equivalent to `1/s`.
  - The corresponding angular frequency in `1/s` is by a factor of `2π` higher.

In `Pint 0.17` however, `Hertz` is equivalent to `1 rad/second`  instead of `1 cycle/second`. 

```
>>> import pint
>>> u = pint.UnitRegistry()
>>> u["1 Hz"].to("rad/second")
<Quantity(1.0, 'radian / second')>
```

**Comment by jules-ch** [comment]
Closed in #961
